jerry-main/main-unix.c

```diff
@@ -687,6 +687,10 @@ main (int argc,
     ret_code = JERRY_STANDALONE_EXIT_CODE_FAIL;
   }
 
+  /* Send signal to client, for the general program byte code */
+#ifdef JERRY_DEBUGGER
+  jerry_debugger_send_type (JERRY_DEBUGGER_PARSE_FUNCTION);
```

Comment:
We really shouldn't need to send this low level message here. If we need, there is some bug in the code.